packages/build/package.json

```diff
@@ -117,6 +117,7 @@
     "is-ci": "^3.0.0",
     "moize": "^6.0.0",
     "nock": "^11.9.1",
+    "@netlify/nock-udp": "JGAntunes/nock-udp.git#main",
```

Comment:
Should this be moved this to `github.com/netlify/nock-udp` and be released to npm?

Reply:
I was planning to create this PR as draft so that it wouldn't get acidentally merged exactly because of this 😅 

> Should this be moved this to github.com/netlify/nock-udp and be released to npm?

Definitely, just wanted to make sure we were going with this approach before doing it 👍